src/micromamba/config.cpp

```diff
@@ -184,6 +184,44 @@ set_config_describe_command(CLI::App* subcom)
     });
 }
 
+void
+set_config_get_command(CLI::App* subcom)
+{
+    auto& config = Configuration::instance();
+
+    auto& key = config.insert(Configurable("key", std::string())
+                                .group("cli")
+                                .rc_configurable(false)
+                                .description("Writes a new value for a key"));
+    subcom->add_option(
+        "key", key.set_cli_config({}), key.description());
+    YAML::Emitter out;
+
+    auto& specs = config.at("config_specs").get_wrapped<std::vector<std::string>>();
+    subcom->add_option("specs", specs.set_cli_config({}), specs.description());
```

Comment:
Also, if i understood correctly this might be too much to be here, right? Should I create an init config?

Reply:
no necessarily, depends if this init will be shared with another sub command or not